importer/EeEt.py

```diff
@@ -49,9 +49,8 @@ def update_heritage(self):
 
     def set_heritage_id(self):
         """Set Estonian cultural monument ID (P2948)."""
-        qual_url = {"described_at_url": self.registrant_url}
         register_no = str(self.registrant_url.split("=")[-1])
```

Comment:
think you can just use `self.number` here

Reply:
Done.